docs/src/components/SearchInput/Results.component.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { ListItem, ListTitle, Typography } from '@retailmenot/anchor';
+
+interface Values {
+    [key: string]: any;
+}
+
+interface ResultsProps {
+    index: number;
+    currentIndex: number;
+    label: string;
+    value?: Values;
+}
+
+export const Results = ({
+    index,
+    currentIndex,
+    label,
+    value,
+}: ResultsProps) => {
+    const hasLink = value ? value.url : false;
```

Comment:
You can then do this logic, not by whether it has a link but by the `listItemType`

Reply:
Just based on behavior and debugging on the doc site, it doesn't appear that `listItemType` gets passed to a custom result template. Bears investigating with the `AutoComplete` component. Until then I'm going to keep the current logic.